CONTRIBUTING.md

```diff
@@ -1,19 +1,52 @@
-# Contribute to rtMedia
+# Contributing to rtMedia
+We love your input! We want to make contributing to rtMedia as easy and transparent as possible, whether it's:
 
-This guide details how to use issues and pull requests to improve rtMedia.
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
 
 ## Closing policy for issues
-
 Github issue-tracker is used **only** for developer & contributor discussion.
-
 For support-request/feature-request, please use - https://rtmedia.io/support/
 
-## Translation
+## Our Development Process
+We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
 
-Please do NOT send pull-request for translations.
+## Flow
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `pre-release`.
+2. If you've added code that should be tested, add tests.
+3. Make sure to follow [WordPress coding standards](https://github.com/WordPress/WordPress-Coding-Standards).
+4. Issue that pull request!
+
+Please include your wordpress.org username while sending a pull-request so that we can add you to the contributor's list.
+
+## Report bugs using Github's [issues](https://github.com/rtmediawp/rtmedia/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
 
+## Write bug reports with detail
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can.
+- What you expected would happen
+- What actually happens
+- Notes/Screenshots (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
```

Comment:
It would be great if you can request a WordPress.org username to add in contributor list.